analyzers/src/SonarAnalyzer.CSharp/Rules/MemberInitializerRedundant.cs

```diff
@@ -220,7 +351,6 @@ public void Report(SyntaxNodeAnalysisContext c, TypeDeclarationSyntax declaratio
                         new DeclarationTuple<IPropertySymbol>
                         {
                             Initializer = p.Initializer,
-                            SemanticModel = semanticModel,
```

Comment:
Please add a simple constructor to the `DeclarationTuple` to remove these painful constructions. They make the code very hard to read (lot of useless lines)